src/tools/rustfmt/src/parse/session.rs

```diff
@@ -23,7 +23,7 @@ use crate::{Config, ErrorKind, FileName};
 
 /// ParseSess holds structs necessary for constructing a parser.
 pub(crate) struct ParseSess {
-    parse_sess: RawParseSess,
+    raw_psess: RawParseSess, // njn: hmm
```

Comment:
no issues with this, but I did chuckle a bit as I can imagine seeing this again in a couple months/years and saying "hmmm" myself in response to the comment 😆

Reply:
Oh, that isn't supposed to be there :)  I use `njn` comments (my initials) to indicate things I need to reconsider/fix up before I post the PR. I wasn't sure about this field name at first but now I think it's fine. I have now updated and removed the comment.